pkg/config/piped.go

```diff
@@ -326,6 +326,8 @@ type PipedGit struct {
 	SSHKeyFile string `json:"sshKeyFile,omitempty"`
 	// Base64 encoded string of ssh-key.
 	SSHKeyData string `json:"sshKeyData,omitempty"`
+	// Whether to create a new branch or not when commit changes in event watcher.
```

Comment:
```suggestion
	// Whether to create a new branch or not when event watcher commits changes.
```

Reply:
thanks! I fixed this comment ! [add createPullRequest flag to event watcher config instead of git config](https://github.com/pipe-cd/pipecd/pull/4395/commits/9f132b7eac0895e9e40dc9d230dd635b3888742d)